Jamroot.jam

```diff
@@ -0,0 +1,20 @@
+path-constant LIBTORRENT_ROOT : . ;
+
+rule asio_src ( properties * )
+{
+	local result ;
+
+	if ! <define>TORRENT_BUILDING_STATIC in $(properties)
```

Comment:
a more straight forward way of doing this (which I would be surprised if it has different behavior) would be:

if <link>static in $(properties)